application/views/admin/survey/subview/accordion/_generaloptions_panel.php

```diff
@@ -316,7 +316,7 @@ function checkSelect2Languages(mylangs) {
         </div>
         <!-- Template -->
 
-        <div class="mb-3">
+        <div class="mb-3" style="height: 300px;">
```

Comment:
No way to have auto height ? I use different size for skelvanilla in 3 and 5 : it work good. 

I know i must redo whole skelvanilla :)

![Capture d’écran du 2023-02-02 12-03-57](https://user-images.githubusercontent.com/1439428/216307992-bd6a7b73-2b42-4f40-936d-0c12a24b7c7b.png)